#31/sample.rb

```diff
@@ -0,0 +1,9 @@
+class Author < ApplicationRecord
+  has_many :books
+end
+
+class Book < ApplicationRecord
+end
+
+author = Author.find_by(name: "Shoheyhey")
+Book.create!(author: author, title: "Hello, World!")
```

Comment:
associationで追加されるメソッドを利用して、以下のように書き換えてもらえないでしょうか？

```suggestion
author.books.create!(title: "Hello, World!")
```